### PR TITLE
Add troubleshooting and known limitations for Plugwise

### PR DESCRIPTION
**Pull request by CoMPaTech**
## Proposed change
This continues on from https://github.com/home-assistant/home-assistant.io/pull/36087 and (too big https://github.com/home-assistant/home-assistant.io/pull/36333) effort

- [x] docs-known-limitations - added as a section, ingesting existing remarks and added new
- [x] docs-troubleshooting - added as a section, ingesting existing

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a "Troubleshooting" section in the Plugwise integration documentation with step-by-step guides for common issues.
	- Added a "Known limitations" section detailing constraints related to schedule configuration and device functionalities.
	- Included subsections on accessing local devices, adjusting update intervals, retrieving diagnostic data, and rebooting the gateway.

### CODE REVIEW

**Comment by CoMPaTech** [comment]
cc @bouwew

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -266,3 +266,72 @@ Stretch (power switches):
...
+
+## Troubleshooting
+
+### Accessing the local device
+
+If you need to configure the gateway directly, without using the Plugwise App, you can find the link to your device by:
+
+1. Go to {% my integrations title="**Settings** > **Devices & services**" %}, and select your integration.
+2. If you have more than one Plugwise gateway, select the one to configure.
+3. Select the gateway device, this should be called 'Adam' or contain 'Smile' in its name.
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: Let's mention the Stretch here too.

File: source/_integrations/plugwise.markdown [hunk 2]
```diff
@@ -266,3 +266,72 @@ Stretch (power switches):
...
+
+{% include common-tasks/define_custom_polling.md %}
+
+### Diagnostic data
+
+If you need to create an issue to report a bug or want to inspect diagnostic data, use the below method to retrieve diagnostics:
+
+1. Go to {% my integrations title="**Settings** > **Devices & services**" %}, and select your integration.
+2. If you have more than one Plugwise gateway, select the gateway that is experiencing issues.
+3. Select the gateway device, this should be called 'Adam' or contain 'Smile' in its name.
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: Add Stretch

File: source/_integrations/plugwise.markdown [hunk 3]
```diff
@@ -266,3 +266,72 @@ Stretch (power switches):
...
+
+## Known limitations
+
+### Schedule configuration and pre-requisites
+
+Creation, modification or deleting of climate schedules is not supported through this integration. We recommend using the Plugwise App or visit the local device to configure schedules. See [accessing the local device](#accessing-the-local-device) above on how to access the local device from Home Assistant.
+
+To display your schedule as a valid `select` option for this integration, ensure that the schedule has a minimal of two schedule points.
+
+### Anna as a zone thermostat
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: `### Anna connected to an Adam`

File: source/_integrations/plugwise.markdown [hunk 4]
```diff
@@ -266,3 +266,72 @@ Stretch (power switches):
...
+
+If you are using your Anna as part of your adam zone control system, it cannot be configured as a smart thermostat. The integration will not discover your Anna or allow manual configuration.
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: adam -> Adam

File: source/_integrations/plugwise.markdown [hunk 5]
```diff
@@ -266,3 +266,72 @@ Stretch (power switches):
...
+
+The change in cooling mode should be detected by Home Assistant. If not, please try to **reload** the Plugwise integration as indicated below and report your findings.
+
+1. Create an issue including your [diagnostic data](#diagnostic-data).
+2. Go to {% my integrations title="**Settings** > **Devices & services**" %}, and select your integration.
+3. On the "**Hubs**" page, use the {% icon "mdi:dots-vertical" %} icon next to your Anna and choose "**Reload**".
+
+### Vacation preset
+
+The `vacation` preset is only available on an Anna.
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: Maybe add here: Adam has a Vacation-mode (called Action in the Plugwise App) that disables the active schedule and sets the vacation-preset for all zones.

Also, there's a Pause-mode that disables the active schedule and sets the away-preset for all zones.

File: source/_integrations/plugwise.markdown (hunk 3 above)
Comment (CoMPaTech) [reply source/_integrations/plugwise.markdown]: Additionally clarified the contents of the section

File: source/_integrations/plugwise.markdown (hunk 4 above)
Comment (CoMPaTech) [reply source/_integrations/plugwise.markdown]: Well spotted

**Review by bouwew — APPROVED** [review]
LGTM!

**Review by silamon — APPROVED** [review]

**Pull request merged**